src/components/navbar/index.js

```diff
@@ -10,29 +11,41 @@ import './navbar.scss';
 
 const Navbar = () => {
     const [showLinks, setShowLinks] = useState(false);
-    const [showCart, setShowCart] = useState(false)
+    const [showCart, setShowCart] = useState(false);
 
+    const controlCart = () => {
+        if (!showLinks) {
+            setShowCart(!showCart);
+        }
+    }
+
+    const controlHamburger = () => {
```

Comment:
`nit:` rename to `toggleHamburger`

**Note:**
The `nit` is just a minor difference of opinion that doesn't really matter, so you can easily just say you like your way better and resolve the comment.